doc/source/ray-observability/reference/system-metrics.rst

```diff
@@ -65,6 +65,9 @@ Ray exports a number of system metrics, which provide introspection into the sta
    * - `ray_component_cpu_percentage`
      - `Component`, `InstanceId`
      - The measured CPU percentage, broken down by logical Ray component. Ray components consist of system components (e.g., raylet, gcs, dashboard, or agent) and the method names of running tasks/actors.
+   * - `ray_component_num_fds`
```

Comment:
I don't think we need to add it here because it is not officially a part of built in dashboard